Zilon.Core/Zilon.Bot.Players.DevelopmentTests/AutoplayEngine.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+
+using Zilon.Bot.Sdk;
+using Zilon.Core.Tactics;
+using Zilon.Emulation.Common;
+
+namespace Zilon.Bot.Players.DevelopmentTests
+{
+    class AutoplayEngine<T> : AutoplayEngineBase<T> where T : IPluggableActorTaskSource
+    {
+        private readonly Startup _startup;
+
+        public AutoplayEngine(Startup startup, BotSettings botSettings) : base(botSettings)
+        {
+            _startup = startup;
+        }
+
+        protected override void ConfigBotAux()
+        {
+            _startup.ConfigureAux(ServiceScope.ServiceProvider);
+        }
+
+        protected override void CatchActorTaskExecutionException(ActorTaskExecutionException exception)
+        {
+            throw exception;
+        }
+
+        protected override void CatchException(Exception exception)
+        {
+            throw exception;
+        }
+
+        protected override void ProcessEnd()
+        {
+            // В тестх не требуется пост-обратоки результатов.
+            // Здесь должна быть фиксация итогов - очки, причина смерти, состояние персонажа на момент смерти.
+        }
+
+        protected override void ProcessSectorExit()
+        {
+            Console.WriteLine("Exit");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5028977)